core/src/main/resources/hudson/PluginWrapper/uninstall.groovy

```diff
@@ -16,7 +16,7 @@ l.layout(permission: Jenkins.ADMINISTER) {
         }
         p { raw _("msg",my.shortName) }
         f.form(method:"post",action:"doUninstall") {
-            f.submit(value:_("Yes"))
+            f.submit(value:_("Yes"), secondary: true, class: "jenkins-button--destructive")
```

Comment:
Is the expectation here really that plugin developers copy & paste code like this into their plugin so it integrates nicely with the UI? Shouldn't this kind of thing be offered as a reusable element, perhaps with custom attributes that are easy to understand and internally translate it into whatever CSS classes you want?

Reply:
IMO a standard button is too simple to be packaged as a component, its just a combination of its content/class/click event. If the button had other functionality, eg animations/complex states/UI, then I'd definitely be in support of making it its own component, such as the help button component.

Happy to be convinced otherwise though/I'm good either way 👍